src/EventListener/RequestConfigurationSubscriber.php

```diff
@@ -91,4 +91,15 @@ public function onKernelController(ControllerEvent $event): void
         $request->setRequestFormat('json');
         $request->attributes->set(self::ATTRIBUTE_API, true);
     }
+
+	private function getAnnotation(\ReflectionMethod $reflection, string $annotationClass)
+	{
+		if (\PHP_VERSION_ID >= 80000) {
```

Comment:
Using PHP 8 does not oblige you to use attributes, it might be better just make attributes take precedence over annotations?

Reply:
prioritization will occur when both attributes and annotations are used at the same time.  I suppose native attributes should take more priority